hack/arktos-up.sh

```diff
@@ -59,6 +59,10 @@ if [[ ! -e "${CONTAINERD_SOCK_PATH}" ]]; then
   exit 1
 fi
 
+echo "Ensuring firewall to allow traffic forward by default"
```

Comment:
Can you add reasons why this is the right way to solve #380?  And also is #380 a bug or an issue of CNI configuration?

Reply:
The PR description was modified to include the source of information the fix is based on. In short, the bridge traffic also traversed iptables rules.

This seems a CNI configuration related environment issue.

Reply:
suggest moving this piece to the inside of "install_cni_bridge". It's probably more clear and won't impact host settings if we don't use beidges.

Reply:
will update as suggested.